src/api/job.js

```diff
@@ -26,4 +26,25 @@ router.post('/', async(request, response) => {
     }
 });
 
+router.put('/:id', async (request, response) => {
+    try {
+        const id = request.params.id;
+        if (!uuidValidate(id)) {
+            return response.status(400).json({message: 'Invalid request.'});
+        }
+        const job = await Job.update(id, request.body);
```

Comment:
I think we should check if the job exists before updating or deleting

Reply:
I fix it. please check again. thanks